google-cloud-logging/src/main/java/com/google/cloud/logging/MonitoredResourceUtil.java

```diff
@@ -161,6 +179,17 @@ private static String getValue(Label label) {
       case NamespaceId:
         value = MetadataConfig.getNamespaceId();
         break;
+      case NamespaceName:
+        try {
+          value =
+              new String(
+                  Files.readAllBytes(
+                      Paths.get("/var/run/secrets/kubernetes.io/serviceaccount/namespace")));
+        } catch (IOException e) {
```

Comment:
There must be no absolute path in the code

Reply:
In Kubernetes environment namespace should be accessible through this path for reference please see [here](https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod).

Can i write like below.

```
private static final String NAMESPACE_PATH="/var/run/secrets/kubernetes.io/serviceaccount/namespace";

case NamespaceName:
    try {
     value =new String(Files.readAllBytes(
                      Paths.get(MonitoredResourceUtil.NAMESPACE_PATH)));
     } catch (IOException e) {
          throw new LoggingException(e, true);
     }
     break;

Reply:
Using constant doesn't remove using the absolute path. 
This is a part of the client library which will be running on the client machine, which could be a Windows machine.
Or k8s could be installed to a different location.

Reply:
I have also try with ```System.getenv("NAMESPACE");```,but using this it's does not fill namespace name inside stackdriver,So
I prefer retrieve it from file.

Reply:
The problem with your approach of reading from the file, the code will not work in many cases.